package.json

```diff
@@ -64,6 +64,7 @@
     "husky": "^9.0.11",
     "jest": "^29.7.0",
     "lint-staged": "^15.2.2",
+    "playwright": "^1.51.1",
```

Comment:
Needed to install latest playwright instead of depending on a transient dep due to an issue installing OS [dependencies on the latest ubuntu](https://github.com/microsoft/playwright/issues/30368) which is fixed in version 1.51.1